apps/web/cypress/tests/notifications-editor.spec.ts

```diff
@@ -120,6 +120,51 @@ describe('Notifications Creator', function () {
       cy.getByTestId('trigger-snippet-btn').click();
       cy.location('pathname').should('equal', '/templates');
     });
+    it('should create multiline in-app notification, send it and receive', function () {
+      waitLoadTemplatePage(() => {
+        cy.visit('/templates/create');
+      });
+      cy.getByTestId('title').type('Test Notification Title');
+      cy.getByTestId('description').type('This is a test description for a test title');
+      cy.get('body').click();
+
+      addAndEditChannel('inApp');
+
+      // put the multiline notification message
+      cy.getByTestId('in-app-editor-content-input')
+        .type('{{firstName}} someone assigned you to {{taskName}}', {
+          parseSpecialCharSequences: false,
+        })
+        .type('{enter}Please check it.');
+      cy.getByTestId('inAppRedirect').type('/example/test');
+      cy.getByTestId('submit-btn').click();
+
+      cy.getByTestId('trigger-snippet-btn').click();
+
+      // trigger the notification
+      cy.eventsTrigger(
+        {
+          name: 'test-notification-title',
+          to: {
+            subscriberId: this.session.user.id,
+            email: this.session.user.email,
+          },
+          payload: {},
+        },
+        this.session.environment.apiKeys[0].key
+      ).then(() => {
```

Comment:
Following up my previous comment this can be used as:

```
 cy.task('createNotifications', {
    identifier: 'test-notification-title',
    token: this.session.environment.apiKeys[0].key,
    count: 1,
    organizationId: this.session.organization._id,
    environmentId: this.session.environment._id,
  });

```

Do you think it will be  ok here?